src/MOI/callbacks/interface.jl

```diff
@@ -0,0 +1,157 @@
+# States
+@enum(
+    CallbackState,
+    CS_NONE,
+    CS_GENERIC,
+    # MOI Callbacks
+    CS_MOI_LAZY_CONSTRAINT,
+    CS_MOI_USER_CUT,
+    CS_MOI_HEURISTIC,
+    # Xpress Callbacks
+    CS_XPRS_OPTNODE,
+    CS_XPRS_PREINTSOL,
+    CS_XPRS_MESSAGE,
+)
+
+function callback_state end
+
+function state_callback end
+
+# Callback Types
+@enum(
+    CallbackType,
+    # MathOptInteface
+    CT_MOI_GENERIC, # temporary
```

Comment:
why?

Reply:
This was some bridge code to migrate from the old interface to the new one. It it gone now.